catroid/src/main/res/values-fr/strings.xml

```diff
@@ -1760,7 +1760,7 @@ needs read and write access to it. You can always change permissions through you
     <string name="formula_editor_logic_comparison">opérateurs de comparaison</string>
     <string name="formula_editor_object_general">propriétés générales</string>
     <string name="formula_editor_object_movement">propriétés de mouvement</string>
-    <string name="formula_editor_device_visual_detection">Visual sensors</string>
+    <string name="formula_editor_device_face_detection">Visual sensors</string>
```

Comment:
undo this changes, nothing to do here, it will eventually be overwritten by the translations